runtime/doc/syntax.txt

```diff
@@ -2278,7 +2278,7 @@ By default mail.vim synchronises syntax to 100 lines before the first
 displayed line.  If you have a slow machine, and generally deal with emails
 with short headers, you can change this to a smaller value: >
 
-    :let mail_minlines = 30
```

Comment:
This doesn't seem related to the rest of the patch, was it intentional?

Reply:
It's not related to the makefile fixes, but it was wrongly aligned.